intl/msg_hash_us.h

```diff
@@ -2733,22 +2733,22 @@ MSG_HASH(MENU_ENUM_SUBLABEL_TAKE_SCREENSHOT,
 #ifdef HAVE_DYNAMIC
 MSG_HASH(
       MENU_ENUM_SUBLABEL_CLOSE_CONTENT,
-      "Closes the current game and application. Any unsaved changes might be lost."
+      "Closes the current content. Any unsaved changes might be lost."
       )
 #else
 MSG_HASH(
       MENU_ENUM_SUBLABEL_CLOSE_CONTENT,
-      "Closes the current game. Any unsaved changes might be lost."
+      "Closes the current content. Any unsaved changes might be lost."
       )
 #endif
```

Comment:
Not squashing them makes it easier to find and review the real changes.

But yes, should be merged.